crates/sui-graphql-rpc/src/types/stake.rs

```diff
@@ -56,6 +67,8 @@ impl Stake {
             Ok(None)
         }
     }
+
+    /// The Stake object as a Move object
```

Comment:
nit: Worth drawing attention to the corresponding Move type here.

```suggestion
    /// The corresponding StakedSui Move object
```